hazelcast/core.py

```diff
@@ -14,34 +17,34 @@ class MemberInfo:
 
     __slots__ = ("address", "uuid", "attributes", "lite_member", "version", "address_map")
 
-    def __init__(self, address, uuid, attributes, lite_member, version, _, address_map):
-        self.address = address
+    def __init__(self, address, member_uuid, attributes, lite_member, version, _, address_map):
+        self.address: Address = address
```

Comment:
Adding type hints to instance variables instead of method parameters doesn't work as well. In the current code, PyCharm doesn't show an "expected type" warning when I pass `1` in the `address`, but the following works as expected:
```
def __init__(self, address: "Address", ...
```

Reply:
Ok, I have changed this, and any of the public APIs that matches with the following regex
```
self\..*:\s\w*\s=
```